daemon/signals.c

```diff
@@ -100,6 +122,76 @@ void signals_reset(void) {
         if(sigaction(signals_waiting[i].signo, &sa, NULL) == -1)
             error("SIGNAL: Failed to reset signal handler for: %s", signals_waiting[i].name);
     }
+
+    if (reaper_enabled == 1)
+        myp_free();
+}
+
+// reap_child reaps the child identified by pid.
+static void reap_child(pid_t pid) {
+    siginfo_t i;
+
+    errno = 0;
+    info("SIGNAL: Reaping pid: %d...", pid);
```

Comment:
Same as below, I think we should remove the prints.

Reply:
How about switching to debug, as they may be useful if there's a bad plugin or something?

Reply:
Sure, you can switch to debug.